spec.html

```diff
@@ -10,7 +10,7 @@
   }
 </script>
 <pre class=metadata>
-  title: ECMAScript® 2016 Language Specification
```

Comment:
I'm not sure if you're asking about the 'title' property in general, or just my change to it.